django_project/frontend/src/containers/MainPage/Data/index.tsx

```diff
@@ -0,0 +1,245 @@
+        { field: 'notes', headerName: 'Notes', width: 150 },
+    ];
+
+    const rows = data.map((data, index) => {
+        const { taxon, property, annualpopulation, annualpopulation_per_activity } = data;
+        return {
+            id: index + 1,
+            property_name: property?.name,
+            property_id: property?.id,
+            owner: property?.owner,
+            owner_email: property?.owner_email,
+            property_type: property?.property_type,
+            province: property?.province,
+            size: property?.size,
+            scientific_name: taxon?.scientific_name,
+            common_name: taxon?.common_name_varbatim,
+            count_total: annualpopulation?.total,
+            count_adult_males: annualpopulation?.adult_male,
+            count_adult_females: annualpopulation?.adult_female,
+            count_subadult_total: annualpopulation?.sub_adult_total,
+            count_subadult_male: annualpopulation?.sub_adult_male,
+            count_subadult_female: annualpopulation?.sub_adult_female,
+            count_juvenile_total: annualpopulation?.juvenile_total,
+            count_juvenile_male: annualpopulation?.juvenile_male,
+            count_juvenile_female: annualpopulation?.juvenile_female,
+            groups: annualpopulation?.group,
+            open_closed_system: annualpopulation?.open_close_system_name,
+            area_available_ha: property?.area_available,
+            ar_name: '',
+            count_method: annualpopulation?.count_method?.name,
+            survey_method: annualpopulation?.survey_method?.name,
+            sampling_effort_coverage: annualpopulation?.sampling_effort,
+            sampling_notes: annualpopulation?.note,
+            count_year: annualpopulation?.year,
+            presence_only: '',
+            reintroduction_total: '',
+            reintroduction_adult_males: '',
+            reintroduction_adult_females: '',
+            reintroduction_male_juveniles: '',
+            reintroduction_female_juveniles: '',
+            founder_population: annualpopulation_per_activity?.founder_population,
+            reintroduction_source: annualpopulation_per_activity?.reintroduction_source,
+            intake_permit_number: '',
+            offtake_total: '',
+            offtake_adult_males: '',
+            offtake_adult_females: '',
+            offtake_male_juveniles: '',
+            offtake_female_juveniles: '',
+            offtake_event: '',
+            additional_detail: '',
+            translocation_destination: '',
+            offtake_permit_number: '',
+            notes: '',
+        };
+    });
+
+    const fetchDataList = () => {
+        setLoading(true)
+        axios.get(`${FETCH_AVAILABLE_DATA}?month=${selectedMonths}&start_year=${startYear}&end_year=${endYear}&species=${speciesParam}`).then((response) => {
+            setLoading(false)
+            if (response.data) {
```

Comment:
If there's no data, the data should be empty, not retain the old one.

Reply:
@dimasciput I checked and i think there is an empty table retains when there is no data available